packages/platform-ios/src/commands/runIOS/index.ts

```diff
@@ -83,7 +83,8 @@ function runIOS(_: Array<string>, ctx: Config, args: FlagsT) {
     }),
   );
 
-  if (devices.length === 0) {
+  // first device is usually a host Mac
```

Comment:
"usually"? What are the cases when it's not?

Reply:
Is it invalid to run on host Mac? My understanding is that it's just another Mac device. It's not a simulator, but a real environment in this case.

Reply:
I believe the first device will always be the host mac which is running the script. I don't think Apple would likely change the order.